lib/Service/CardService.php

```diff
@@ -351,6 +354,29 @@ public function update($id, $title, $stackId, $type, $owner, $description = '',
 
 
 		$card = $this->cardMapper->update($card);
+
+		$stack = $this->stackMapper->find($card->getStackId());
+		$board = $this->boardService->find($stack->getBoardId());
```

Comment:
```suggestion
		$board = $this->boardService->find($this->cardMapper->findBoardId($card->getId());
```

We can get the board id directly which is likely cached